proposals/0472-task-start-synchronously-on-caller-context.md

```diff
@@ -3,7 +3,7 @@
 * Proposal: [SE-0472](0472-task-start-synchronously-on-caller-context.md)
 * Authors: [Konrad 'ktoso' Malawski](https://github.com/ktoso)
 * Review Manager: [Tony Allevato](https://github.com/allevato)
-* Status: **Accepted**
+* Status: **Implemented (Swift 6.2)**
 * Implementation: https://github.com/swiftlang/swift/pull/79608
```

Comment:
Might make sense to add https://github.com/swiftlang/swift/pull/81428 and https://github.com/swiftlang/swift/pull/81572 to `Implementation:` section for future reference.